metadata.json

```diff
@@ -49,7 +49,10 @@
     {
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": [
-        "16.04"
+        "16.04",
+        "18.04",
```

Comment:
18 is EoL. I suggest to add only 20 and 22 and drop 16 in another PR

Reply:
we still have 3 months of 18.04 left ^^ but sure, will do so